### PR TITLE
feat(colors): Add card gradients to colors theme 

### PR DESCRIPTION
**Pull request by ChefHutch**
- Add card gradients that are being used more and more in designs to `theme/colors.ts`

(in particular `linear-gradient(111.68deg, #F2ECF2 0%, #E8F2F6 100%)`
Screenshot 2021-04-09 at 14 34 40

- Replace the existing `cardHeaderBackground` with these `theme/colors` values

### CODE REVIEW

File: packages/pancake-uikit/src/__tests__/components/buttonmenu.test.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ it("renders correctly", () => {
 
     .c3 {
       background-color: transparent;
-      color: #1FC7D4;
+      color: #8f80ba;
```
Comment (ChefHutch) [inline packages/pancake-uikit/src/__tests__/components/buttonmenu.test.tsx:138]: Unrelated snapshot fix. Must have gone in before the CI started running tests on PRs.

File: packages/pancake-uikit/src/theme/colors.ts [hunk 2]
```diff
@@ -32,6 +32,11 @@ export const lightColors: Colors = {
   card: "#FFFFFF",
   gradients: {
     bubblegum: "linear-gradient(139.73deg, #E6FDFF 0%, #F3EFFF 100%)",
+    cardBackground: {
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/theme/colors.ts:35]: imo this should ba at the root like bubblegum (generic color)

Comment (ChefHutch) [reply packages/pancake-uikit/src/theme/colors.ts:35]: Sure,

```js
  gradients: {
    bubblegum: "linear-gradient(139.73deg, #313D5C 0%, #3D2A54 100%)",
    cardBackgroundDefault: "linear-gradient(166.77deg, #3B4155 0%, #3A3045 100%)",
    cardBackgroundBlue: "linear-gradient(180deg, #00707F 0%, #19778C 100%)",
    cardBackgroundViolet: "linear-gradient(180deg, #6C4999 0%, #6D4DB2 100%)",
  },
```

Or should we get bugs to come up with some thematic names?

Comment (hachiojidev) [reply packages/pancake-uikit/src/theme/colors.ts:35]: Bubblegum is what is was called in Figma.

File: packages/pancake-uikit/src/theme/types.ts [hunk 3]
```diff
@@ -29,6 +29,11 @@ export type Shadows = {
 
 export type Gradients = {
   bubblegum: string;
+  cardBackground: {
+    default: string;
+    blue: string;
+    violet: string;
+  };
```
Comment (hachiojidev) [inline packages/pancake-uikit/src/theme/types.ts:36]: The `Gradients` object should only be top-level values, i.e. no `cardBackground`. That means however we will need to come up with a name for `default`

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: packages/pancake-uikit/src/theme/types.ts (hunk 3 above)
Comment (ChefHutch) [reply packages/pancake-uikit/src/theme/types.ts:36]: Yea makes sense not to nest them. Let me ping Bugs for some inspiration on 'default'

File: packages/pancake-uikit/src/theme/colors.ts (hunk 2 above)
Comment (ChefHutch) [reply packages/pancake-uikit/src/theme/colors.ts]: Updated this to remove nesting & reflect Figma color scheme

File: packages/pancake-uikit/src/theme/types.ts (hunk 3 above)
Comment (ChefHutch) [reply packages/pancake-uikit/src/theme/types.ts]: Updated this to remove nesting & reflect Figma color scheme

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**